client/src/components/Icons.jsx

```diff
@@ -14,7 +14,6 @@ class Icons extends React.Component {
 		this.openSearch = this.openSearch.bind(this);
 	}
 
-	//onClick function for the menu bars
 	closeNav() {
 		document.getElementById("sideNav").style.width = "0";
```

Comment:
Is there another way to do this on the virtual DOM instead of manipulating the DOM itself? I'm just a bit curious...

Reply:
yeah not sure about that! I was wondering if i should make it a component instead? what do you think?